Makefile

```diff
@@ -38,7 +39,7 @@ docs-lint: ## Check documentation for common syntax errors.
 .PHONY: docs
 docs: ## Build project documentation in live reload for editing
 # Spins up livereload environment for editing; blocks.
-	make -C docs/ clean && sphinx-autobuild docs/ docs/_build/html
+	make -C docs/ clean && sphinx-autobuild `[ $(PWD) = /vagrant ] && echo '--host 0.0.0.0'` docs/ docs/_build/html
```

Comment:
Slick, resolves my concerns about the outside-the-VM workflow mentioned in #2173.